roles/reproducer/defaults/main.yml

```diff
@@ -31,3 +31,8 @@ cifmw_reproducer_dnf_tweaks: []
 cifmw_reproducer_compute_repos: []
 cifmw_reproducer_repositories_path: "src"
 cifmw_reproducer_play_extravars: []
+cifmw_reproducer_supported_hypervisor_os:
+  CentOS:
+    minimum_version: 9
+  RedHat:
+    minimum_version: 9.4
```

Comment:
hm, minimal is probably still 9.2 - we started with that, and "it works". Not sure we want to push 9.4.. Maybe we should stick to the major release ("9")?

Reply:
I would go with either 9.2 or 9.4, I think CentOS Stream 9 is basically unversioned, it's all just 9 but with RHEL we have some granularity so we might as well use it. 

The reason for going with 9.4 is all downstream CI is using it, so it's the "tested" version. Thoughts?